core/src/parquet/read/values.rs

```diff
@@ -455,23 +455,21 @@ macro_rules! make_int_variant_impl {
 
                 while num - i >= 32 {
                     unsafe {
-                        let in_slice = std::slice::from_raw_parts(in_ptr, 32);
```

Comment:
This is unsafe because there is no guarantee that the buffer is properly aligned for u32 typw (in_ptr is defined earier as `let mut in_ptr = &src_data[src_offset..] as *const [u8] as *const u8 as *const u32;`).